Assets/BossRoom/Scripts/Client/UI/Lobby/LobbyUIMediator.cs

```diff
@@ -46,6 +50,24 @@ ClientGameNetPortal clientGameNetPortal
             RegenerateName();
 
             m_ClientNetPortal.NetworkTimedOut += OnNetworkTimeout;
+            m_Subscriptions = serviceError.Subscribe(ServiceErrorHandler);
+        }
+
+        private void ServiceErrorHandler(UnityServiceErrorMessage error)
+        {
+            var errorMessage = error.Message;
+            if (error.AffectedService == UnityServiceErrorMessage.Service.Lobby)
+            {
+                if ((error.OriginalException as LobbyServiceException).Reason == LobbyExceptionReason.LobbyConflict)
+                {
+                    // LobbyConflict can have multiple causes. Let's add other solutions here if there's other situations that arise for this.
+                    errorMessage += "\nSee logs for possible causes and solution.";
+                    Debug.LogError($"Got service error {error.Message} with LobbyConflict. Possible conflict cause: Trying to play with two builds on the " +
+                              $"same machine. Please use command line arg '{ClientMainMenuState.AuthProfileCommandLineArg} someName' to set a different auth profile.\n");
+                }
+            }
+
+            PopupPanel.ShowPopupPanel("Service error", errorMessage);
```

Comment:
Should this be moved to a separate script, like with ConnectionStatusMessageUIManager? Either way, that would also resolve [MTT-2734](https://jira.unity3d.com/browse/MTT-2734) I think

Reply:
Yeah you're right, I thought at first to make it lobby specific, but this applies to all service errors actually... let me look

Reply:
done